aggregator/aggregator_mock.go

```diff
@@ -18,340 +18,396 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// Automatically generated by MockGen. DO NOT EDIT!
+// Code generated by MockGen. DO NOT EDIT.
 // Source: github.com/m3db/m3aggregator/aggregator (interfaces: ElectionManager,FlushTimesManager,PeriodicFlusher,PlacementManager,Handler,Writer)
 
 package aggregator
 
 import (
 	"context"
+	"reflect"
 	"time"
 
 	"github.com/m3db/m3aggregator/generated/proto/flush"
 	"github.com/m3db/m3cluster/placement"
 	"github.com/m3db/m3cluster/shard"
 	"github.com/m3db/m3metrics/metric/aggregated"
 	"github.com/m3db/m3x/watch"
+	"github.com/uber-go/tally"
```

Comment:
Hey so this is not ordered properly due to an issue I fixed in the latest rev of ci-scripts, can you pull that in and re-run mockgen? Thanks.

Reply:
sounds good